packages/graphile-build-pg/src/plugins/PgConnectionTotalCount.js

```diff
@@ -7,25 +7,18 @@ export default (function PgConnectionTotalCount(builder) {
     (fields, build, context) => {
       const {
         extend,
-        inflection,
         graphql: { GraphQLInt, GraphQLNonNull },
         pgSql: sql,
       } = build;
       const {
-        scope: { isPgRowConnectionType, pgIntrospection: table },
+        scope: { isPgRowConnectionType, nodeType },
         fieldWithHooks,
         Self,
       } = context;
 
-      if (
-        !isPgRowConnectionType ||
-        !table ||
-        table.kind !== "class" ||
-        !table.namespace
-      ) {
+      if (!isPgRowConnectionType || !nodeType || !nodeType.name) {
```

Comment:
I think this is a breaking change for people using `makeExtendSchemaPlugin` to add their own connections:

https://github.com/graphile/graphile-engine/blob/b5d9e99fefa0af027bfcaeae06443eff00412aea/packages/graphile-utils/src/makeExtendSchemaPlugin.ts#L721-L725

Please can you support the old `pgIntrospection` in addition to `nodeType`?

Reply:
Good catch. Done. https://github.com/graphile/graphile-engine/pull/529/commits/a75c4d7a462df01bdb83a193eb01d55ff8698537